### ISSUE TITLE
Service principal script fails

### ISSUE THREAD

**Issue by pierceboggan** [MicrosoftDocs/power-platform#256]
To use the service principal script, I first had to run the following PowerShell commands:

* `Install-Module -Name AzureAD -AllowClobber -Scope CurrentUser`
* `Install-Module -Name Microsoft.Xrm.OnlineManagementAPI -AllowClobber - Scope CurrentUser`
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fba2155b-98b3-64e0-29f5-79cf2b1a90f1
* Version Independent ID: d2b30089-7cf3-563a-eb38-d112793beb9b
* Content: [Microsoft Power Platform Build Tools for Azure DevOps - Power Platform](https://docs.microsoft.com/en-us/power-platform/alm/devops-build-tools#build-tool-tasks)
* Content Source: [power-platform/alm/devops-build-tools.md](https://github.com/MicrosoftDocs/power-platform/blob/master/power-platform/alm/devops-build-tools.md)
* Service: **powerapps**
* GitHub Login: @phecke
* Microsoft Alias: **pehecke**

**Comment by Altair115** [comment]
Even after following the steps above i still keep getting the following error.

```
\New-CrmServicePrincipal.ps1 : A parameter cannot be found that matches parameter name 'TenantId'.
At line:1 char:1
+ .\New-CrmServicePrincipal.ps1 -TenantLocation "Europe"
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [New-CrmServicePrincipal.ps1], ParameterBindingException
    + FullyQualifiedErrorId : NamedParameterNotFound,New-CrmServicePrincipal.ps1
```

**Comment by scottdurow** [comment]
The fix for this is to use version 1.2.0.1 of Microsoft.Xrm.OnlineManagementAPI:
```
Uninstall-Module -Name Microsoft.Xrm.OnlineManagementAPI
Install-Module -Name Microsoft.Xrm.OnlineManagementAPI -MaximumVersion 1.2.0.1
```

**Comment by phecke** [comment]
@pierceboggan Thanks for your feedback. We are closing this issue because we are either unable to find relevant information or reproduce this. If you're still experiencing the issue described here, please reopen or create a new issue.
